### ISSUE TITLE
Two tests failing.

### ISSUE THREAD

**Issue by jeremyjbowers** [newsdev/nyt-ap-elections#95]
```
E..............E..........
======================================================================
ERROR: Failure: DistributionNotFound (The 'nyt-ap-elections' distribution was not found and is required by the application)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/jbowers/src/nyt/nyt-ap-elections/elex/cli/__init__.py", line 1, in <module>
    from app import main
  File "/Users/jbowers/src/nyt/nyt-ap-elections/elex/cli/app.py", line 9, in <module>
    VERSION = pkg_resources.get_distribution('nyt-ap-elections').version
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 551, in get_distribution
    dist = get_provider(dist)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 431, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 952, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 839, in resolve
    raise DistributionNotFound(req, requirers)
DistributionNotFound: The 'nyt-ap-elections' distribution was not found and is required by the application

======================================================================
ERROR: Failure: DistributionNotFound (The 'nyt-ap-elections' distribution was not found and is required by the application)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/jbowers/src/nyt/nyt-ap-elections/tests/test_cli.py", line 8, in <module>
    from elex.cli.app import ElexApp
  File "/Users/jbowers/src/nyt/nyt-ap-elections/elex/cli/__init__.py", line 1, in <module>
    from app import main
  File "/Users/jbowers/src/nyt/nyt-ap-elections/elex/cli/app.py", line 9, in <module>
    VERSION = pkg_resources.get_distribution('nyt-ap-elections').version
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 551, in get_distribution
    dist = get_provider(dist)
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 431, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 952, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/jbowers/.virtualenvs/nyt-ap-elections/lib/python2.7/site-packages/pkg_resources/__init__.py", line 839, in resolve
    raise DistributionNotFound(req, requirers)
DistributionNotFound: The 'nyt-ap-elections' distribution was not found and is required by the application

----------------------------------------------------------------------
Ran 26 tests in 15.214s

FAILED (errors=2)
```

**Comment by eads** [comment]
Interesting... not failing on this end.

**Comment by eads** [comment]
is this happening on a pypi install or dev version?

**Comment by jeremyjbowers** [comment]
Well, all working fine now? No idea what changed.

**Comment by eads** [comment]
Yeah, weird. Maybe setuptools version and the real version were out of sync.

On Thu, Nov 19, 2015 at 11:44 AM, Jeremy Bowers <notifications@github.com>
wrote:

> Closed #95 <https://github.com/newsdev/nyt-ap-elections/issues/95>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/newsdev/nyt-ap-elections/issues/95#event-469383834>.
>



-- 
David Eads | http://recoveredfactory.net

"Medical statistics will be our standard of measurement: we will weigh life
for life and see where the dead lie thicker, among the workers or among the
privileged." -- Rudolf Virchow
